spring-web-modules/spring-mvc-basics-4/src/main/java/com/baeldung/controller/controller/GreetingsController.java

```diff
@@ -15,7 +15,7 @@ public class GreetingsController {
     
     @GetMapping(value = "/greetings-with-response-body", produces="application/json")
     public String getGreetingWhileReturnTypeIsString() { 
-        return "{\"test\": \"Hello using @ResponseBody\"}";
```

Comment:
Should we keep the previous body, or else we need to change the mapping as well since it mentions "-with-response-body"

Reply:
I will change the body into `Hello using @ResponseBody from the annotation @RestController`.